v1/hwevent/event.go

```diff
@@ -0,0 +1,111 @@
+// Copyright 2020 The Cloud Native Events Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package hwevent
+
+import (
+	"encoding/json"
+	"fmt"
+
+	log "github.com/sirupsen/logrus"
+
+	cloudevents "github.com/cloudevents/sdk-go/v2"
+	"github.com/google/uuid"
+	"github.com/redhat-cne/sdk-go/pkg/channel"
+	"github.com/redhat-cne/sdk-go/pkg/pubsub"
+
+	"github.com/redhat-cne/sdk-go/pkg/hwevent"
+)
+
+//PublishCloudEventToLog .. publish event data to a log
+func PublishCloudEventToLog(e cloudevents.Event) {
+	log.Infof("Publishing event to log %#v", e)
+}
+
+//CloudNativeEvent gets Cloud Native Event object
+func CloudNativeEvent() hwevent.Event {
+	return hwevent.Event{Type: "HwEvent"}
```

Comment:
We should change this to use Enum eventually

Reply:
added TODO